dirty_deploy.bat

```diff
@@ -1 +1,6 @@
+rem test set: jUnit tests for java code
+rem code quality checks: no
+rem deploy: deploy on local server tomcat 7
+rem intent of using: development
```

Comment:
We have two identical intent. How I can (as a developer) decide which file is better to use? Please add some more description here (and possibly in other files)